checks.c

```diff
@@ -1798,8 +1798,10 @@ static void check_graph_child_address(struct check *c, struct dt_info *dti,
 		struct property *prop = get_property(child, "reg");
 
 		/* No error if we have any non-zero unit address */
-		if (prop && propval_cell(prop) != 0)
+		/* We have a check for val.len == sizeof(cell_t) elsewhere */
```

Comment:
If that's the case, can't you move that above this point and then you could safely use `propval_cell`

Reply:
> Edit: `check_graph_port()` and `check_graph_endpoint()` are now prerequisites of `check_graph_child_address()` and `propval_cell` is used again.

That sounds like the right solution: this is exactly the sort of thing that the prerequisite system is for.